typescript/cli/src/utils/time.ts

```diff
@@ -1,6 +1,12 @@
 export function getTimestampForFilename() {
+  const pad = (n: number) =>
+    n.toLocaleString('en-US', { minimumIntegerDigits: 2, useGrouping: false });
   const now = new Date();
-  return `${now.getFullYear()}-${
-    now.getMonth() + 1
-  }-${now.getDate()}-${now.getHours()}-${now.getMinutes()}-${now.getSeconds()}`;
+  const year = now.getFullYear();
+  const month = pad(now.getMonth() + 1);
+  const date = pad(now.getDate());
+  const hours = pad(now.getHours());
+  const minutes = pad(now.getMinutes());
+  const seconds = pad(now.getSeconds());
+  return `${year}-${month}-${date}-${hours}-${minutes}-${seconds}`;
```

Comment:
do we really need seconds? :P

Reply:
At first I didn't include it but the in rare cases, like fully recovered deployments, we would overwrite other files